docs/usage/models.md

```diff
@@ -29,33 +29,38 @@ class User(BaseModel):
     name: str = 'Jane Doe'
 ```
 
-`User` here is a model with two fields `id` which is an integer and is required,
-and `name` which is a string and is not required (it has a default value). The type of `name` is inferred from the
-default value, and so a type annotation is not required (however note [this](#field-ordering) warning about field
-order when some fields do not have type annotations).
+In this example, `User` is a model with two fields:
+
+* `id`, which is an integer and is required
+* `name`, which is a string and is not required (it has a default value).
+
+The type of `name` is inferred from the
+default value, and so a type annotation is not required (however, please see the warning on
+[Field Ordering](#field-ordering) regarding field order when some fields do not have type annotations).
```

Comment:
```suggestion
```

I think let's just get rid of this comment now, as it no longer applies (in v2, all fields _must_ have annotations).